tests/unit/data_masking/test_data_masking.py

```diff
@@ -0,0 +1,247 @@
+import copy
+import json
+
+import pytest
+from itsdangerous.url_safe import URLSafeSerializer
+
+from aws_lambda_powertools.shared.constants import DATA_MASKING_STRING
+from aws_lambda_powertools.utilities.data_masking.base import DataMasking
+from aws_lambda_powertools.utilities.data_masking.provider import Provider
+
+
+class MyEncryptionProvider(Provider):
```

Comment:
I can pull out all the input data necessary for this into a utilities file if that would be cleaner